commit e853648f9ff0682f0b217de541d96f398509ff0b
Author: Sean Young <sean@mess.org>
Date:   Wed Nov 9 14:13:33 2016 -0200

    ir-keytable: make it possible to select the rc5 streamzap variant
    
    It was not possible to select the rc-5-sz protocol.
    
    Signed-off-by: Sean Young <sean@mess.org>
    Signed-off-by: Mauro Carvalho Chehab <mchehab@s-opensource.com>

diff --git a/utils/keytable/keytable.c b/utils/keytable/keytable.c
index 3922ad2..202610a 100644
--- a/utils/keytable/keytable.c
+++ b/utils/keytable/keytable.c
@@ -123,6 +123,7 @@ const struct protocol_map_entry protocol_map[] = {
 	{ "rc5",	NULL,		SYSFS_RC5	},
 	{ "rc-5x",	NULL,		SYSFS_INVALID	},
 	{ "rc5x",	NULL,		SYSFS_INVALID	},
+	{ "rc-5-sz",	NULL,		SYSFS_RC5_SZ	},
 	{ "jvc",	"/jvc_decoder",	SYSFS_JVC	},
 	{ "sony",	"/sony_decoder",SYSFS_SONY	},
 	{ "sony12",	NULL,		SYSFS_INVALID	},
